html-exercise/index.html

```diff
@@ -9,354 +8,504 @@
 
     <meta name="description" content="Fashion, " />
     <meta name="keywords" content="Pant, Shirt, FreeShipping" />
-    <link rel="stylesheet" href="./styles.css" />
 
-    <link rel="stylesheet"
-      href="https://pro.fontawesome.com/releases/v5.15.4/css/all.css?fbclid=IwAR34A7kjQKkOfVF7Ad8tcx0t0dvEUt7MntsBu5PhGIXfMgV1V45PYpIYK4M" />
+    <link rel="stylesheet" href="./assets/css/base.css" />
+    <link rel="stylesheet" href="./assets/css/common.css" />
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;0,600;0,700;1,600&display=swap"
+      rel="stylesheet"
+    />
   </head>
 
   <body>
     <header>
       <div class="container">
-        <h1 class="header-logo">
-          <a class="title">
-            <img class="image" src="./assets/images/social-media-icon/logo.svg" alt="" />
-            E-SHOP
+        <div class="d-flex justify-content-between align-items-center">
+          <h1 class="header-logo d-flex align-items-center">
```

Comment:
Em ko add thêm nhiều class ở đây khi đã có define `header-logo` dùng để style rồi, nếu đã có class riêng css thì mọi thuộc tính đều add ở css, ko add thêm class, check lại toàn bộ file html